src/core/Akka.Cluster.Tests.MultiNode/LeaderLeavingSpec.cs

```diff
@@ -99,7 +99,7 @@ public void
                         EnterBarrier("leader-left");
 
                         // verify that the LEADER is EXITING
-                        exitingLatch.Ready(TestLatch.DefaultTimeout);
+                        exitingLatch.Ready(TestKitSettings.DefaultTimeout);
```

Comment:
Just wanted to get rid of an `OBSOLETE` warning while I was poking around in the test suite